cardano-node/src/Cardano/Node/Tracing/Era/Shelley.hs

```diff
@@ -248,22 +250,22 @@ instance ( ShelleyBasedEra era
          , ToJSON (Core.AuxiliaryDataHash (Ledger.EraCrypto era))
          , LogFormatting (PredicateFailure (ShelleyUTXO era))
          , LogFormatting (PredicateFailure (ShelleyUTXOW era))
-         , LogFormatting (PredicateFailure (Core.EraRule "DELEGS" era))
+      --   , LogFormatting (PredicateFailure (Core.EraRule "DELEGS" era))
          , LogFormatting (PredicateFailure (Core.EraRule "UTXOW" era))
          ) => LogFormatting (ShelleyLedgerPredFailure era) where
   forMachine dtal (UtxowFailure f)  = forMachine dtal f
-  forMachine dtal (DelegsFailure f) = forMachine dtal f
+  forMachine _dtal (DelegsFailure _f) = error "TODO" -- forMachine dtal f
```

Comment:
Let's not error here, this will bring down the node.

Reply:
Fixed